src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import { PropTypes } from 'prop-types';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const possiblePartner = sex === 'm' ? 'wife' : 'husband';
```

Comment:
you could place each property and condition on new line for better readibility.

Reply:
agree